### PR TITLE
Updates uniforms for naval officers

### PR DESCRIPTION
**Pull request by Ty-Omaha**
## What Does This PR Do
Updates sprites for naval officers, also adds armor to their uniform and changes their station to NAS Trurl instead of the random "NCV FEARLESS"

## Why It's Good For The Game
So people dont confuse NTR/MAGI/BS as naval officers and it looks good
Credit to Ionward for sprites

image

image

image

:cl: Ty-Omaha and Ionward
tweak: Updates Naval Officer uniforms, new sprite, armor, and tweaked description.
/ 🆑

### CODE REVIEW

File: code/modules/clothing/under/miscellaneous.dm [hunk 1]
```diff
@@ -96,20 +96,21 @@
...
 /obj/item/clothing/under/rank/centcom/officer
-	desc = "Gold trim on space-black cloth, this uniform displays the rank of \"Lieutenant-Commander\" and bears \"N.C.V. Fearless CV-286\" on the left shoulder."
-	name = "\improper Nanotrasen Officers Uniform"
-	icon_state = "officer"
-	item_state = "g_suit"
-	item_color = "officer"
+	desc = "Gold trim on space-black cloth, this uniform displays the rank of \"Lieutenant-Commander\" and bears \"N.A.S. Trurl \" on the left shoulder. Worn exclusively by officers of the Nanotrasen Navy. It's got exotic materials for protection."
+	name = "\improper Nanotrasen Naval Officers Uniform"
+	icon_state = "navy_gold"
+	item_state = "navy_gold"
```
Comment (TDSSS) [inline code/modules/clothing/under/miscellaneous.dm]: duplicate line

**Comment by TDSSS** [comment]
👀 those reverts


but looks good and seems like a nice solution to the confusion problem. 👍

**Comment by Ty-Omaha** [comment]
sorry there's like a million commits

**Comment by Carthusia** [comment]
Is nice, I still don't think there is a confusion issue, but this does look like a nicer uniform for a higher ranking officer, more gold and trim like effect to it, and a seperation of the higher ranks from the lower ranks adds a nice bit of detail.

**Comment by Ty-Omaha** [comment]
well this replace the sprite for all naval officers, there's no captain vs officer version anymore

**Comment by Carthusia** [comment]
I dono, seems like it would be nice to have both. 

A captain and an Ensign board the station but the Captain has the same uniform as the lower ranking officer, instead of the more flashy one compared to the ensign.

At a glance if each has a different uniform, its easier to determine which is the higher rank, often by the amount of gold and trim they have. 

Such being the case even in modern times with dress uniforms in north america and historically further back.

**Comment by Ty-Omaha** [comment]
yeah but naval officers are pretty much the same anyways and very rare to see on station. they'll have different berets but the chance of having **both** a naval officer and a naval captain onboard station is virtually non existent

**Comment by Carthusia** [comment]
Fluff as it may be, its still interesting fluff.

Possible situations that mean lead to 2 officers/their escorts being aboard, while rare, are still possible and have been done from what i can recall.

It's just a nice side bonus to have various clothing items to pick for the scenario being created at the discretion of the one making it.

**Comment by Ty-Omaha** [comment]
they can be distinguished from each other by their berets

**Comment by Carthusia** [comment]
not 100% sure what VV means, can you elaborate?

My guess is it means they can spawn anything, which if its the case then you are correct so long as the old uniform is still available via this method.

**Comment by Ty-Omaha** [comment]
view variables - they can change the uniform sprite to anything they want even a random file on their computer

**Comment by Carthusia** [comment]
Ooohh, wasn't aware, that's pretty sweet. That does explain all the insane things i've seen over time, like Ian being hyper realistic and looking downright terrifying because of it or the singularity being a JPG of a banana.

File: code/modules/clothing/under/miscellaneous.dm [hunk 2]
```diff
@@ -96,21 +96,24 @@
...
-	item_state = "g_suit"
-	item_color = "officer"
+	desc = "Gold trim on space-black cloth, this uniform displays the rank of \"Lieutenant-Commander\" and bears \"N.A.S. Trurl \" on the left shoulder. Worn exclusively by officers of the Nanotrasen Navy. It's got exotic materials for protection."
+	name = "\improper Nanotrasen Naval Officer Uniform"
+	icon_state = "navy_gold"
+	item_state = "navy_gold"
+	item_color = "navy_gold"
 	displays_id = 0
 	flags_size = ONESIZEFITSALL
+	armor = list("melee" = 60, "bullet" = 60, "laser" = 60,"energy" = 60, "bomb" = 60, "bio" = 100, "rad" = 30, "fire" = 60, "acid" = 100)
```
Comment (Kyep) [inline code/modules/clothing/under/miscellaneous.dm]: Not sure about the armor. Will this stack with specops gear, giving specops officers near-100 damage reduction? Unsure.

**Comment by Ty-Omaha** [comment]
@Kyep removed armor ready for merge @variableundefined

**Comment by Kyep** [comment]
RIP me getting confused for a NT Rep every time I appear on the bridge.

**Pull request merged**